internal/admin/admin.go

```diff
@@ -703,7 +703,8 @@ func (adm *Admin) updateLogStream(ctx context.Context, lsid types.LogStreamID, p
 
 	lsrmd, err := adm.snmgr.AddLogStreamReplica(ctx, pushedReplica.GetStorageNodeID(), newLSDesc.TopicID, lsid, pushedReplica.GetStorageNodePath())
 	if err != nil {
-		return nil, errors.Wrap(err, "update log stream")
+		// TODO:: handle with error code
+		return nil, status.Errorf(codes.Unavailable, "update log stream: add logstream replica %s", err.Error())
```

Comment:
AddLogStreamReplica RPC returns a gRPC error codified with ResourceExhausted. Sorry for the undocumented error code.

Could you keep the error code like the code below?
```go
code := status.Code(err)
if code != codes.ResourceExhausted {
    code = codes.Unavailable
}
return nil, status.Errorf(code, "add log stream: %s", err.Error())
```
Ignore this suggestion if all errors returned from the storage node should be Unavailable.

Reply:
Is It already codified all? Is Unavailable correct in this case?

https://github.com/kakao/varlog/blob/00c0c42292cc38eab45e12ede28116564f6b26b9/internal/storagenode/storagenode.go#L354

Reply:
Anyway, should I consider only the case of ResourceExhausted?

Reply:
Currently, only the ResourceExhausted error is returned. I will add gRPC errors to the storage node RPCs later.